src/solver.rs

```diff
@@ -216,12 +215,7 @@ impl<I: std::ops::Fn(Epoch, SV, usize) -> Option<InterpolationResult>> Solver<I>
         // Regularly refer to https://github.com/nyx-space/anise/blob/master/data/ci_config.dhall for the latest CRC, although it should not change between minor versions!
         // NB: a default almanac will soon be provided by ANISE directly
         //     this triggers a network access at least once
-        let almanac = Almanac::default()
-            .load_from_metafile(MetaFile {
-                uri: "http://public-data.nyxspace.com/anise/v0.4/pck08.pca".to_string(),
-                crc32: Some(3072159656), // Specifying the CRC allows only downloading the data once.
-            })
-            .map_err(Error::Almanac)?;
+        let almanac = Almanac::until_2035().map_err(Error::Almanac)?;
```

Comment:
This will load _more_ data than just the PCK (planetary constants kernel) but it does not require internet at runtime, which I think is one of your requirements.